### ISSUE TITLE
Hi, is the project frozen? 

### ISSUE THREAD

**Issue by SExperiaMini** [fnzv/net-Shield#5]
I can not install, if you answer - I am ready to provide errors

**Comment by SExperiaMini** [comment]
Reading package lists... Done
Building dependency tree
Reading state information... Done
ipset is already the newest version (6.29-1).
software-properties-common is already the newest version (0.96.20.8).
jq is already the newest version (1.5+dfsg-1ubuntu0.1).
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
4 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up nginx-core (1.10.3-0ubuntu0.16.04.3) ...
Setting up nginx (1.10.3-0ubuntu0.16.04.3) ...
Setting up python3-certbot-nginx (0.28.0-1+ubuntu16.04.1+certbot+3) ...
Setting up python-certbot-nginx (0.28.0-1+ubuntu16.04.1+certbot+3) ...
gpg: keyring `/tmp/tmpwnfvcknm/secring.gpg' created
gpg: keyring `/tmp/tmpwnfvcknm/pubring.gpg' created
gpg: requesting key 75BCA694 from hkp server keyserver.ubuntu.com
gpg: /tmp/tmpwnfvcknm/trustdb.gpg: trustdb created
gpg: key 75BCA694: public key "Launchpad PPA for certbot" imported
gpg: Total number processed: 1
gpg:               imported: 1  (RSA: 1)
OK
Hit:1 http://archive.canonical.com/ubuntu xenial InRelease
Hit:2 http://security.ubuntu.com/ubuntu xenial-security InRelease
Hit:3 http://archive.ubuntu.com/ubuntu xenial InRelease
Hit:4 http://ppa.launchpad.net/certbot/certbot/ubuntu xenial InRelease
Hit:5 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
python-certbot-nginx is already the newest version (0.28.0-1+ubuntu16.04.1+certb                                                                                                                                                             ot+3).
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
Setting ipt new log file..
Installing Nginx for nShield proxy..\n
Reading package lists... Done
Building dependency tree
Reading state information... Done
nginx is already the newest version (1.10.3-0ubuntu0.16.04.3).
0 upgraded, 0 newly installed, 0 to remove and 5 not upgraded.
Copying example configuration... \n
--2019-01-22 10:11:13--  https://raw.githubusercontent.com/fnzv/nShield/master/e                                                                                                                                                             xample/nshield.conf
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133                                                                                                                                                             , 151.101.64.133, 151.101.128.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.13                                                                                                                                                             3|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 157 [text/plain]
Saving to: '/etc/nshield/nshield.conf'

/etc/nshield/nshiel 100%[===================>]     157  --.-KB/s    in 0s

2019-01-22 10:11:13 (32.8 MB/s) - '/etc/nshield/nshield.conf' saved [157/157]

/etc/nginx/snippet/ssl.conf: No such file or directory
/etc/nginx/snippet/le.conf: No such file or directory
Enter a domain to protect: mysite.com
Enter the real IP address of the website: 1.1.1.1.1
Compiling source code..
import cycle not allowed
package main
        imports github.com/BurntSushi/toml
        imports bufio
        imports bytes
        imports errors
        imports runtime
        imports internal/bytealg
        imports internal/cpu
        imports runtime
import cycle not allowed
package main
        imports github.com/BurntSushi/toml
        imports bufio
        imports bytes
        imports errors
        imports runtime
        imports internal/bytealg
        imports internal/cpu
        imports runtime
cp: cannot stat 'config-shield': No such file or directory
cp: cannot stat 'shield': No such file or directory

**Comment by SExperiaMini** [comment]
git project - and = bash install.sh
---------------------
ubuntu-16.04

**Comment by SExperiaMini** [comment]
> Hello!
> Could you tell me what commands did you run and your current operative system?
> 
> The project isn't on active development or production ready but there is a remote possibility i'll release new features / make it production ready

Запустил, можете закрывать тикет.
А еще вопрос, есть какие нибудь правила которые отбивают атаки или банят? Потому что на тесте выявилось что при атаке сервер точно так же в отключке

**Comment by fnzv** [comment]
Yep this is normal, when you finished the installation you need to run the command:

> config-nshield

And then the nginx proxy will be configured (you can see the configuration under /etc/nshield/nshield.conf)


Then if you open the nginx configuration you will see the proxy configured:

> cat /etc/nginx/sites-enabled/d*


To check for bans you can open the log file:

> tailf  /var/log/nshield.log

**Comment by SExperiaMini** [comment]
> Yep this is normal, when you finished the installation you need to run the command:
> 
> > config-nshield
> 
> And then the nginx proxy will be configured (you can see the configuration under /etc/nshield/nshield.conf)
> 
> Then if you open the nginx configuration you will see the proxy configured:
> 
> > cat /etc/nginx/sites-enabled/d*
> 
> To check for bans you can open the log file:
> 
> > tailf  /var/log/nshield.log

Maybe something is not configured, but everything seems to be working, logs are being written - everything is active, but when I check, the server drops from one IP address

**Comment by fnzv** [comment]
If you can reach your site the proxy is working correctly, to check for bans try to make a large amount of requests and then check your logs or iptables-save/ipset list for updated entries

**Comment by SExperiaMini** [comment]
> Если вы можете зайти на свой сайт, прокси-сервер работает правильно, чтобы проверить наличие банов, попробуйте сделать большое количество запросов, а затем проверьте свои журналы или список iptables-save / ipset для обновленных записей.

The fact is that if I start testing the site again on DDoS - the server will drop again for a day.
That's why I want to find out what my problem could be, or the point in the script and it does not ban me right away.
Let's in order to clean the server you need to install:
ngnix, iptables, golang.
And then configure the server and run the install script?

**Comment by fnzv** [comment]
You can see what the script does here:
https://github.com/fnzv/net-Shield/blob/master/install.sh

TLDR:
If you want to cleanup remove nginx jq software-properties-common ipset golang 

also:
Rsyslog: /etc/rsyslog.d/10-ipt.conf & iptables logging (just an entry)
Crontab: rule to run nshield

**Comment by SExperiaMini** [comment]
> You can see what the script does here:
> https://github.com/fnzv/net-Shield/blob/master/install.sh
> 
> TLDR:
> If you want to cleanup remove nginx jq software-properties-common ipset golang
> 
> also:
> Rsyslog: /etc/rsyslog.d/10-ipt.conf & iptables logging (just an entry)
> Crontab: rule to run nshield

I have not been able to understand the principle of your script for 3 days.
With installed programs, when installing the script and starting the server it falls, as it is not observed in the work Nginx
But according to the logs, everything is supposedly super and works
Could you make a video manual for people like me.
Installations from the start > minimal ubuntu 16.04 etc

**Comment by SExperiaMini** [comment]
> Hello!
> This is the quick start video:
> https://asciinema.org/a/zozehdooPDbvem9tCDLI321Hp

Yes, I watched it
Is it put on a clean assembly? no nginx, go lang, etc?
